node/src/types/block.rs

```diff
@@ -1089,6 +1090,43 @@ pub(crate) mod json_compatibility {
     }
 }
 
+/// Finality signature that can be gossiped between nodes or sent to clients.
```

Comment:
Maybe:
`A validator's signature of a block, to confirm it is finalized. Clients and joining nodes should wait until the signers' combined weight exceeds their fault tolerance threshold before accepting the block as finalized.`

Reply:
Done in [`4e187e2` (#635)](https://github.com/CasperLabs/casper-node/pull/635/commits/4e187e27d16153b0f933af336f0abae1cdb7263c)